Script/build_packages.py

```diff
@@ -75,7 +77,7 @@ def refresh_staging():
     return staging_path
 
 def extract_into(destdir, zippath):
-    with zipfile.ZipFile(zippath, "r") as z:
+    with contextlib.closing(zipfile.ZipFile(zippath, "r")) as z:
```

Comment:
Nevermind, I was just getting annoyed with older version of python interfering when I run the script without specifying correct interpreter version.